### PR TITLE
[incubator/logstash] update to logstash version 6.3.2

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**: updates logstash to version 6.3.2**

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/logstash/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ updateStrategy:
 
 image:
   repository: docker.elastic.co/logstash/logstash-oss
-  tag: 6.3.1
+  tag: 6.3.2
```
Comment (davidkarlsen) [inline incubator/logstash/values.yaml:11]: could you update the README to be consistent with the values.yaml?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by monotek** [comment]
@davidkarlsen 
sure, done :-)

**Pull request merged**